extensions/mssql/src/main.ts

```diff
@@ -134,6 +134,55 @@ export async function activate(context: vscode.ExtensionContext): Promise<MssqlE
 	context.subscriptions.push(new DeleteFilesCommand(prompter, appContext));
 	context.subscriptions.push({ dispose: () => languageClient.stop() });
 
+	azdata.ui.registerModelViewProvider('bdc-endpoints', async (view) => {
+
+		const endpointsArray: Array<Utils.IEndpoint> = view.serverInfo.options['clusterEndpoints'];
+		const managementProxyEp = view.serverInfo.options['clusterEndpoints'].find(e => e.serviceName === 'management-proxy');
+		if (managementProxyEp) {
+			endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Grafana Dashboard', '/grafana'));
+			endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Kibana Dashboard', '/kibana'));
+		}
+
+		const gatewayEp = view.serverInfo.options['clusterEndpoints'].find(e => e.serviceName === 'gateway');
+		if (gatewayEp) {
+			endpointsArray.push(getCustomEndpoint(gatewayEp, 'Spark History', '/gateway/default/sparkhistory'));
+			endpointsArray.push(getCustomEndpoint(gatewayEp, 'Yarn History', '/gateway/default/yarn'));
+		}
+
+		const container = view.modelBuilder.flexContainer().withLayout({ flexFlow: 'column', width: '100%', height: '100%', alignItems: 'left' }).component();
+		const endpointsContainer = view.modelBuilder.flexContainer().withLayout({ flexFlow: 'column', width: '100%', height: '100%', alignItems: 'left' }).component();
+		endpointsArray.forEach(endpointInfo => {
+			const endPointRow = view.modelBuilder.flexContainer().withLayout({ flexFlow: 'row' }).component();
+			const nameCell = view.modelBuilder.text().withProperties<azdata.TextComponentProperties>({ value: localize('serviceEndpoint.' + endpointInfo.serviceName, '{0}', endpointInfo.serviceName) }).component();
```

Comment:
Localized IDs need to be predefined. In this case, we have to use the unlocalized values returned (since it's pure data representation of what's in the connected SQL Server). Please do not use localized here, and maybe add a comment explaining why

Reply:
Sure. Charles commented on my last changes about localizing them and I thought may be that's required. Will remove the localization.